### PR TITLE
added a blog post about the wechaty-log-monitor plugin I just wrote

### PR DESCRIPTION
**Pull request by ghost**
I just wrote a wechaty plugin for log-related devops in wechaty. https://github.com/archywillhe/wechaty-log-monitor  

Here is a blog post about it and how it can be used for operations like "QR Rescue".

---
Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.

- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to the blog `/download/2018` directory.
- [x] Add `<!--more-->` section to just show the abstract in blog homepage.
- [x] Add teaser image for the blog
- [x] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.github.io/blob/master/_data/authors.yml> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

[See more](https://github.com/wechaty/wechaty.github.io#contribute-guideline)

### CODE REVIEW

File: _posts/2020-08-09-wechaty-log-monitor.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
+---
+title: "Chatbots二重奏：用wechaty-log-monitor插件实现「掉线给码」等DevOps"
+author: archy
+date: "2020-08-09 20:20:00 +0800"
+categories: project
+tags:
+  - plugin
+  - wechaty
+  - devops
+image: /assets/2020/wechaty-log-monitor/demo.jepg
```
Comment (lijiarui) [inline _posts/2020-08-09-wechaty-log-monitor.md]: should be `demo.jpeg`

File: _posts/2020-08-09-wechaty-log-monitor.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+> [![flair](https://camo.githubusercontent.com/c551a231a6cda28e59291fa091ddcb7b9899f6ec/68747470733a2f2f737461636b65786368616e67652e636f6d2f75736572732f666c6169722f313334303435332e706e67)](https://stackoverflow.com/users/2041954/%E5%90%96%E5%A5%87%E8%AF%B4-%E4%BD%95%E9%AD%8F%E5%A5%87archy-will-he)
+>
+
+最近基于Wechaty[做的一个学中文小助手ARCHY开始营业了](https://mp.weixin.qq.com/s/FcgaOOnZNPUuMSihmMs_lw)🤖🤖🍜～
+
+为了能让小助手可以有更棒的 readily availability，我写了这个[wechaty-log-monitor插件](https://github.com/archywillhe/wechaty-log-monitor)来给在production跑的Wechaty做日志相关的devops。这是一个建立于两个chatbots的二重奏。
+
+目前插件的主要功能是「掉线给码」：一个Wechaty掉线了，另一个Wechaty会发QR码给这个Wechaty的微信号来重新登陆。
+
+![qr-rescue](/assets/2020/wechaty-log-monitor/demo.jepg)
```
Comment (lijiarui) [inline _posts/2020-08-09-wechaty-log-monitor.md]: demo.jpeg

File: _posts/2020-08-09-wechaty-log-monitor.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+botBob.use(WechatyLogMonitor({
+  logOperations:[qrResuceForA]
+}))
+```
+
+就几行代码，就能更轻松地进行掉线重登的developer operation了。 提高readily availability：只要两个没有同时掉线，另一个就能迅速得救！
+
+p.s. 掉线后，bot发一个二维码就不会再发给你了。如果你想要最新的登陆二维码，发「qr」给bot就行。
+
+![qr](/assets/2020/wechaty-log-monitor/qr.jepg)
```
Comment (lijiarui) [inline _posts/2020-08-09-wechaty-log-monitor.md]: should be `qr.jpeg`

File: _posts/2020-08-09-wechaty-log-monitor.md [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+}
+```
+
+未来如果要发展到把项目整个production相关的DevOps（不单只是WeChaty相关的Operations，如在production跑的MongoDB相关的Operations、Restful API服务器相关的Operations等）都运用WeChaty来给团队塑造一个简单、容易上手的流程，也就是把chatbot变成了一个简易的terminal，那时候我们可以设定让更危险的Operations变得需要短信验证码、authy等方式去做Authentication。
+
+我相信Auth功能对于wechaty-log-monitor插件来说将会是一个有意思的发展方向之一。
+
+若pragmatically，『chatbot变成一个更简易的terminal』这件事真的行得通，那另一个非常有意思的发展方向就是结合OpenAI提及到的GPT3的一个很有意思的应用：[Natural Language Shell](https://beta.openai.com/?app=productivity&example=4_2_0) - 运用自然语言去做执行unix等命令。
+
+![qr-rescue](/assets/2020/wechaty-log-monitor/gtp3.jeg)
```
Comment (lijiarui) [inline _posts/2020-08-09-wechaty-log-monitor.md]: should be jpg

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request.

**Comment by lijiarui** [comment]
Since we change the template or wechaty blog, we have a break change:

## Before
You should add your profile at `author.yml`

## After

### 1. Create a file at `_developers` named as `yourname.md` 

 `yourname` should be the same as the author name you put in the blog:
```
---
title: XX
date: XX
author: yourname
categories: XX
tags:
  - XX
image: XX
---
```
### 2. The file should begin with:
```
---
name: Jiarui LI (李佳芮)
site: https://lijiarui.github.io
bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
avatar: /assets/developers/lijiarui/avatar.png
email: rui@juzi.bot
twitter: https://weibo.com/u/2175505900
---
```

### 3. Example:

- lijiarui.md: https://github.com/wechaty/wechaty.github.io/blob/master/_developers/lijiarui.md
- It will shown like this on wechaty blog: https://wechaty.github.io/developers/lijiarui/

Looking forward to your profile file at the `_developers` folder!

**Comment by ghost** [comment]
Oooh! Got it! 👍 Great refurbishment you guys are doing for the website! =) 

Also thanks for proofreading! Sorry for the Freudian slips on the file format! Will get them fixed!

& Will set up my profile and let you know again 😎

**Comment by ghost** [comment]
alright! I think it's good to go!

File: _posts/2020-08-09-wechaty-log-monitor.md [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+> 作者: [Archy Will He 何魏奇](https://github.com/archywillhe/)，functional programmer, interested in computational semantics，目前在全职做[吖奇说(ARCHY.SH)](https://archy.sh)这个项目。Working with GPT-2 (and hopefully with 3 soon!)
+>
+> Github Repo: [wechaty-log-monitor plugin](https://github.com/archywillhe/wechaty-log-monitor)
+>
+> [![flair](https://camo.githubusercontent.com/c551a231a6cda28e59291fa091ddcb7b9899f6ec/68747470733a2f2f737461636b65786368616e67652e636f6d2f75736572732f666c6169722f313334303435332e706e67)](https://stackoverflow.com/users/2041954/%E5%90%96%E5%A5%87%E8%AF%B4-%E4%BD%95%E9%AD%8F%E5%A5%87archy-will-he)
+>
+
+最近基于Wechaty[做的一个学中文小助手ARCHY开始营业了](https://mp.weixin.qq.com/s/FcgaOOnZNPUuMSihmMs_lw)🤖🤖🍜～
+
+![gif-demo](/assets/2020/wechaty-log-monitor/archy-demo.gif)
```
Comment (lijiarui) [inline _posts/2020-08-09-wechaty-log-monitor.md:24]: Where is this file?

**Review by lijiarui — CHANGES_REQUESTED** [review]
Well, it seems not ready...

File: _posts/2020-08-09-wechaty-log-monitor.md (hunk 5 above)
Comment (ghost) [reply _posts/2020-08-09-wechaty-log-monitor.md:24]: aw sorry! Added now!

**Comment by lijiarui** [comment]
Good job!

**Pull request merged**